velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -310,6 +311,16 @@ std::vector<RowVectorPtr> AggregationFuzzerBase::generateInputDataWithRowNumber(
   auto size = vectorFuzzer_.getOptions().vectorSize;
   velox::test::VectorMaker vectorMaker{pool_.get()};
   int64_t rowNumber = 0;
+
+  // Randomly generate the number of partitions
+  auto numPartitions = randInt(1, size / 10);
```

Comment:
What if size < 10?

Reply:
Thanks for pointing out this case, updated the calculation for max number of partitions, could you please take another look?